### PR TITLE
Week 264 by ash

### PR DESCRIPTION
**Pull request by ash**

### CODE REVIEW

File: challenge-264/ash/raku/ch-1.raku [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+sub solve($s) {
+    my $chars = $s.comb.Bag;
+
+    my @candidates = $chars.grep: * ~~ /<[A..Z]>/;
+    @candidates = @candidates.grep({$chars{$_.key.lc}});
+
+    @candidates = @candidates.sort;    
+
+    return @candidates.elems ?? @candidates[*-1].key !! '-';
+}
```
Comment (cono) [inline challenge-264/ash/raku/ch-1.raku:24]: tooo long ;-)

```
raku -e '"PeRlwEeKLy".comb.sort({$^b.lc cmp $^a}).rotor(2 => -1).grep({.[0].lc eq .[1].lc}).first.first.uc.say'
```

File: challenge-264/ash/raku/ch-1a.raku [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+for @s -> $s {
+    say "$s: " ~ solve($s);
+}
+
+sub solve($s) {
+    my @chars = $s.comb;
+
+    my @lc = @chars.grep: * ~~ /<[a..z]>/;
+    my @uc = @chars.grep: * ~~ /<[A..Z]>/;
+    @lc.=map: *.uc;
```
Comment (cono) [inline challenge-264/ash/raku/ch-1a.raku:20]:
```
raku -e 'my @lc = <a b c>; @lc .= uc; @lc.say'
[A B C]
```

File: challenge-264/ash/raku/ch-1a.raku [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    my $x = @lc ∩ @uc; # $x is a Set
+
+    return $x ?? $x.keys.sort[*-1] !! '-';
```
Comment (cono) [inline challenge-264/ash/raku/ch-1a.raku:24]: BAH!
```
"aSds".comb.classify({$_ ~~ /<[a..z]>/ ?? "lc" !! "uc"}).values».uc.reduce(&[∩]).keys.sort[*-1].say
```

Comment (cono) [reply challenge-264/ash/raku/ch-1a.raku:24]: or even shorter:
```
raku -e '"aSds".comb.classify(so * ~~ /<[a..z]>/).values».uc.reduce(&[∩]).keys.sort[*-1].say'
S
```

**Comment by manwar** [comment]
Thanks @ash

**Pull request merged**